scripts/send_application_reminder.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+from datetime import datetime
+from datetime import timedelta
+
+sys.path.insert(1, ".")
+
+from external_services.exceptions import NotificationError
+import api.routes.application.helpers as helpers
+from app import app
+from config import Config
+from db.models.applications import ApplicationsMethods
+from db.models.forms import FormsMethods
+from external_services.models.notification import Notification
+from flask import current_app
+
+
+def application_deadline_reminder(fund_id: str, round_id: str):
+
+    current_date_time = datetime.now()
+
+    fund_rounds = helpers.get_data(
+        Config.FUND_STORE_API_HOST
+        + Config.FUND_ROUND_ENDPOINT.format(fund_id=fund_id, round_id=round_id)
+    )
+    
+    fund_deadline_string = fund_rounds.get("deadline")
+    fund_deadline = datetime.fromisoformat(fund_deadline_string)
+    reminder_date = fund_deadline - timedelta(days=14)
+
+    if reminder_date < current_date_time < fund_deadline:
+
+        status = {
+            "status_only": "IN_PROGRESS",
+            "fund_id": fund_id,
+            "round_id": round_id,
+        }
+
+        in_progress_applications = ApplicationsMethods.search_applications(
+            **status
+        )
+
+        all_applications = []
+        for application in in_progress_applications:
+
+            application["round_name"] = fund_rounds.get("title")
+            account_id = helpers.get_account(
```

Comment:
Can this be called "account" (as it is an account object that's returned not an account_id)?

Reply:
changed to account